msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -485,8 +486,13 @@ void validateConfiguration() {
     }
 
     private void validateRedirectUri(@NonNull final String redirectUri) {
-        final boolean isInvalid = TextUtils.isEmpty(redirectUri) || !hasSchemeAndAuthority(redirectUri);
-
+        boolean isInvalid = false;
+        if (mAppContext.getPackageName().equalsIgnoreCase(AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME))
```

Comment:
We have a. null-safe comparison, we should use it.

Reply:
can you point me to it

Reply:
StringUtil.equalsIgnoreCase or ObjectUtils.equals, I believe.

Reply:
I reversed the comparison to be safer